docs/powergate/cli/pow_data_log.md

```diff
@@ -0,0 +1,30 @@
+## pow data log
```

Comment:
What do you think about switching to `pow data logs`?
At least I have a feeling that other CLI in the work usually use "logs", not "log". 
I'm used to it now in the Powergate CLI because I've mistyped it several times, but curious about your opinion.

Reply:
we can add it as an alias so both will work. I'll create an issue in powergate

Reply:
already made this change actually